mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -354,22 +354,48 @@ object Tool extends App with LazyLogging {
   private def configure(config: ConfigCmd): Unit = {
     config.submitStats match {
       case Some(isEnabled) =>
-        val statCollector = new ExecutionStatisticsCollector()
-        if (isEnabled) {
-          logger.info("Statistics collection is ON.")
-          logger.info("This also enabled TLC and TLA+ Toolbox statistics.")
-          statCollector.set(Selection.ON)
-        } else {
-          logger.info("Statistics collection is OFF.")
-          logger.info("This also disabled TLC and TLA+ Toolbox statistics.")
-          statCollector.set(Selection.NO_ESC)
+        if (configDirExistsOrCreated()) {
+          val statCollector = new ExecutionStatisticsCollector()
+          // protect against potential exceptions in the tla2tools code
+          try {
+            if (isEnabled) {
+              statCollector.set(Selection.ON)
+              logger.info("Statistics collection is ON.")
+              logger.info("This also enabled TLC and TLA+ Toolbox statistics.")
+            } else {
+              statCollector.set(Selection.NO_ESC)
+              logger.info("Statistics collection is OFF.")
+              logger.info("This also disabled TLC and TLA+ Toolbox statistics.")
+            }
+          } catch {
+            case e: Exception =>
+              logger.warn(e.getMessage)
+              logger.warn(s"Unable to update statistics configuration. The other features will keep working.")
+          }
         }
-
       case None =>
         ()
     }
   }
 
+  private def configDirExistsOrCreated(): Boolean = {
+    // A temporary fix for the issue #762: create ~/.tlaplus if it does not exist
+    val configDir = new File(System.getProperty("user.home", ""), ".tlaplus")
+    if (configDir.exists()) {
+      true
+    } else {
+      try {
+        configDir.mkdir()
+        true
+      } catch {
+        case e: Exception =>
+          logger.warn(e.getMessage)
+          logger.warn(s"Unable to create the directory $configDir. The statistics will not be collected.")
```

Comment:
This function is general more general than stat collection. It could end up getting called for other configuration setting/editing, so I think probably better not to put the stats remark in here. Instead, how about we handle that in the else branch of the `submitStats` handler:

```suggestion
          logger.warn(s"Unable to create the directory $configDir.")
```